airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -147,12 +149,13 @@ def get_asset_events(
     source_map_index: Annotated[
         FilterParam[int | None], Depends(filter_param_factory(AssetEvent.source_map_index, int | None))
     ],
+    timestamp_range: Annotated[RangeFilter, Depends(datetime_range_filter_factory("timestamp", AssetEvent))],
```

Comment:
We also need tests for `api_fastapi` endpoint.

Reply:
Are there currently tests for any api_fastapi endpoints?

Reply:
Tests added